components/security/authorization.rst

```diff
@@ -121,10 +121,10 @@ on a "remember-me" cookie, or even authenticated anonymously?
     use Symfony\Component\Security\Core\Authentication\Token\AnonymousToken;
     use Symfony\Component\Security\Core\Authentication\Token\RememberMeToken;
 
-    $anonymousClass = AnonymousToken::class;
-    $rememberMeClass = RememberMeToken::class;
+    $anonymousToken = AnonymousToken::class;
```

Comment:
These should be `$anonymousTokenClass` and `$rememberMeClass` because they are classes, not tokens. Or better yet: why don't we remove these useless variables and moved their values to the class constructor?

Reply:
I thought about that but the point of PR is better naming? I saw some places where I could just move the values but I did not know if it would be appropriate for this PR?